actions/team_actions.jsx

```diff
@@ -105,3 +110,12 @@ export function updateTeamsOrderForUser(teamIds) {
         dispatch(savePreferences(currentUserId, teamOrderPreferences));
     };
 }
+
+export function fetchThreadMentionCountsByChannel() {
+    return async (dispatch, getState) => {
+        const state = getState();
+        if (isCollapsedThreadsEnabled(state)) {
+            dispatch(getThreadMentionCountsByChannel(getCurrentTeamId(state)));
```

Comment:
(cc @reflog in case I'm off on something or missed anything)

- `getThreadMentionCountsByChannel` is done on a per-team basis
- Looks like this is fired on WS `reconnect` and `focusPost`
- it could get it's own state and short circuit internally, but currently all feature-on-off gate logic is outside of `actions/threads.ts`
  
In the end, looks like this team action is being removed in the next PR as further changes to mentions/unreads are underway.
https://github.com/mattermost/mattermost-webapp/pull/7733/files#diff-42c7d37cdea1c44e8773b4f6a75c9c03291f9bffebb8d268186c0a41ff929a3f